[2025-03-12T13:26:30.970+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-12T13:26:31.002+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T13:26:27.288859+00:00 [queued]>
[2025-03-12T13:26:31.021+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T13:26:27.288859+00:00 [queued]>
[2025-03-12T13:26:31.022+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-12T13:26:31.040+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): start_pipeline> on 2025-03-12 13:26:27.288859+00:00
[2025-03-12T13:26:31.047+0000] {standard_task_runner.py:72} INFO - Started process 200 to run task
[2025-03-12T13:26:31.058+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_lake_pipeline', 'start_pipeline', 'manual__2025-03-12T13:26:27.288859+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/tmp/tmpvvuudtkb']
[2025-03-12T13:26:31.064+0000] {standard_task_runner.py:105} INFO - Job 38: Subtask start_pipeline
[2025-03-12T13:26:31.111+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-12T13:26:31.182+0000] {task_command.py:467} INFO - Running <TaskInstance: data_lake_pipeline.start_pipeline manual__2025-03-12T13:26:27.288859+00:00 [running]> on host ***-scheduler
[2025-03-12T13:26:31.364+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_lake_pipeline' AIRFLOW_CTX_TASK_ID='start_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-03-12T13:26:27.288859+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-12T13:26:27.288859+00:00'
[2025-03-12T13:26:31.366+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-12T13:26:31.393+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-12T13:26:31.394+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Pipeline démarré"']
[2025-03-12T13:26:31.412+0000] {subprocess.py:99} INFO - Output:
[2025-03-12T13:26:31.414+0000] {subprocess.py:106} INFO - Pipeline démarré
[2025-03-12T13:26:31.415+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-12T13:26:31.470+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-12T13:26:31.471+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_lake_pipeline, task_id=start_pipeline, run_id=manual__2025-03-12T13:26:27.288859+00:00, execution_date=20250312T132627, start_date=20250312T132631, end_date=20250312T132631
[2025-03-12T13:26:31.559+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-12T13:26:31.595+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-12T13:26:31.600+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
